src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="wrapper">
+        <legend>Personal information</legend>
+
+        <label class="fields">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            maxlength="8"
+          >
+        </label>
+
+        <label class="fields">
+          Name:
+          <input
+          type="text"
+          name="userName"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="userName"
            autocomplete="off"
          >
```
![image](https://user-images.githubusercontent.com/96409563/220445719-b9109521-27c4-4e1c-93c8-c7cee9fbef9c.png)

Reply:
Tag’s closing bracket   (closing-and-opening-   
                should be on the same   on-seme-level)          
                level as opening one
<textarea
            name="feedBack"
            cols="10"
            rows="2"
          ></textarea>